doc/wiki/dashboard-audit.md

```diff
@@ -0,0 +1,668 @@
+## Dashboard query audit
+Reviewing the following Dashboard queries to ensure we're displaying and describing the data correctly:
+
+1. New registered patients
```

Comment:
I would use markdown links here to link to the specific sections: https://www.w3schools.io/file/markdown-links/

Reply:
Niceee thanks for the tip!